### ISSUE TITLE
There is a basic record of attendance!

### ISSUE THREAD

**Issue by mlandauer** [openaustralia/theyvoteforyou#805]
See for example: http://parlinfo.aph.gov.au/parlInfo/search/display/display.w3p;db=CHAMBER;id=chamber%2Fvotes%2F8bca2a7b-a412-4073-809b-520dcc307f5f%2F0019;query=Id%3A%22chamber%2Fvotes%2F8bca2a7b-a412-4073-809b-520dcc307f5f%2F0000%22

**Comment by cofiem** [comment]
I don't know if this is useful or not, and I know this issue is almost 4 years old. I was interested to try this....

Anyway, I created a scraper that captures this attendance information. I wouldn't recommend running the scraper on  morph.io, as it will try to scrape all attendance records available - which go back to about 1973 from what I can see. It would likely time out. It also will currently create duplicate records if it is run more than once.

Be aware these records may be incorrect - I haven't done thorough correctness checks.
Also be aware that the attendance records are for if a senator or member was present at any time during a sitting day - this does not mean the senator or member was present the *whole* day.

Repo is here: https://github.com/cofiem/australia_parliament_attendance

I've run the scraper locally (with a cache of the downloaded pages). I ran it over the first page of results (2018 - 2015). I've committed the sqlite db and a zip file containing the cached pages in the 'cache' directory.

Let me know if this gets used for anything.
